### PR TITLE
Add confirmation button for shutdown, reboot, and logout actions

### PR DESCRIPTION
**Pull request by Fzzr**
This is very primitive. Don't actually merge this yet, it needs maximum CSS. @pchaussalet quick logic check would also be appreciated.

Ticket: #13605

### CODE REVIEW

**Comment by Fzzr** [comment]
Looks Good to me. @joshuarule time for aesthetics, over to you.

File: ui/main-sidebar/main-sidebar.reel/main-sidebar.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ var Component = require("montage/ui/component").Component;
  */
 exports.MainSidebar = Component.specialize({
 
+    confirmationPromise: {
```
Comment (joshuarule) [inline ui/main-sidebar/main-sidebar.reel/main-sidebar.js]: What is this needed for?

Comment (Fzzr) [reply ui/main-sidebar/main-sidebar.reel/main-sidebar.js]: @pchaussalet I think that got replaced with `_confirmationDeferred`, can you confirm?

Comment (pchaussalet) [reply ui/main-sidebar/main-sidebar.reel/main-sidebar.js]: Indeed, it is unused, it has already been cleaned up by @joshuarule.

**Pull request merged**